Docs/slave_extras.md

```diff
@@ -0,0 +1,164 @@
+## Singularity Extras for Mesos Slaves
+
+*This assumes you've already followed the steps from [install.md](install.md)*
+
```

Comment:
Maybe add something here explaining exactly what someone might need to use the Singularity Custom Executor for.